payment_allpay/__init__.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please apply the comments in previous module about copyright, openerp.py and PEP8